sdk/spring/compatibility-tests-job.yml

```diff
@@ -48,5 +48,5 @@ jobs:
           javaHomeOption: 'JDKVersion'
           jdkVersionOption: $(JavaTestVersion)
           jdkArchitectureOption: 'x64'
-          goals: 'clean test -Pdev '
```

Comment:
Why need this?

Reply:
Please check the exception module not found: com.azure.identity.providers.core  https://dev.azure.com/azure-sdk/public/_build/results?buildId=1982270&view=logs&j=37f2ab31-7ccd-5b05-707c-a4d1ecafcba4&t=99b9dd50-aa5d-58ff-8682-743b936e6cdc&l=1359, it will depend on the identify providers core, if not installed, the auto module name will not be generated for jdbc mysql and postgersql.

Reply:
ok, this is because we are using the current version, right?

Reply:
Yes.